src/types/bytes.rs

```diff
@@ -26,6 +26,21 @@ impl PyBytes {
         unsafe { py.from_owned_ptr(ffi::PyBytes_FromStringAndSize(ptr, len)) }
     }
 
+    /// Creates a new Python bytestring object.
+    /// The uninitialised bytestring must be initialised in the `init` closure.
```

Comment:
Maybe specifically say the _entire_ uninitialised bytestring?

Also, if you're up for it, adding a little example would be very much appreciated.

Reply:
Where and in what format should I add an example?

Reply:
I've added examples in 533c105b73d214afdb14033f45b1659cf0723233